README.md

```diff
@@ -1,24 +1,62 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# Micro-Reddit
 
-Things you may want to cover:
+> This is a solution to the [Odin project lesson: Micro-Reddit](https://www.theodinproject.com/courses/ruby-on-rails/lessons/building-with-active-record-ruby-on-rails). 
 
-* Ruby version
+![screenshot](screenshot.png)
 
-* System dependencies
 
-* Configuration
+## Built With
 
-* Database creation
+- Ruby on Rails
+- Ruby
 
-* Database initialization
 
-* How to run the test suite
+## Getting Started 
 
-* Services (job queues, cache servers, search engines, etc.)
+- Run this command in your OS terminal: `git clone https://github.com/disc3110/Miro-reddit.git` to get a copy of the project.
+- Open a terminal in the project folder and run `rails console`.
 
-* Deployment instructions
 
-* ...
+### Prerequisites
+
+* Internet connection
+* Having Ruby installed on you OS
+* Having Rails installed on your OS
+
+
+### Description
```

Comment:
- This section should appear at the start of this page. This will make it easier for someone to know what the project is about at a glance upon opening the repo